src/settings.go

```diff
@@ -68,18 +74,56 @@ func loadUserConfiguration(env environmentInfo) (*Settings, error) {
 		return nil, errors.New("INVALID CONFIG PATH")
 	}
 
-	_, j, err := jsonc.ReadFromFile(settingsFile)
+	config.AddDriver(yaml.Driver)
+	config.AddDriver(json.Driver)
+	config.AddDriver(toml.Driver)
+	config.WithOptions(func(opt *config.Options) {
+		opt.TagName = "config"
+	})
+
+	err := config.LoadFiles(settingsFile)
 	if err != nil {
 		return nil, errors.New("UNABLE TO OPEN CONFIG")
 	}
 
-	err = json.Unmarshal(j, &settings)
+	err = config.BindStruct("", &settings)
 	if err != nil {
 		return nil, errors.New("INVALID CONFIG")
 	}
+
 	return &settings, nil
 }
 
+func exportSettings(settingsFile, format string) string {
+	if len(format) == 0 {
+		format = config.JSON
+	}
+	err := config.LoadFiles(settingsFile)
+	if err != nil {
+		return "UNABLE TO OPEN CONFIG"
```

Comment:
I'm hitting this if I have invalid json, yaml, or toml.  What about an error more like "INVALID CONFIG" in that situation?  For example, if I use a trailing comma in the last element of a json block, or indent too much in yaml, or add a stray `]` in toml.  I also noticed `err` says exactly what the problem was and what line it's on, which could be helpful to the user.  Maybe just `fmt.Println` it?  Just an idea.

Reply:
I'll add the context indeed.

Reply:
done

Reply:
works fine 👍 with 8116c36